code/modules/surgery/organs/tongue.dm

```diff
@@ -75,8 +76,11 @@
 	say_mod = "hisses"
 	taste_sensitivity = 10 // combined nose + tongue, extra sensitive
 	modifies_speech = TRUE
+	languages_native = list(/datum/language/draconic)
 
 /obj/item/organ/tongue/lizard/handle_speech(datum/source, list/speech_args)
+	if(owner.get_selected_language() in languages_native)
+		return //to keep using autohiss with a native language, a player can use prefixes without using the language as main one
```

Comment:
Could be a good idea to put this to tongue's handle_speech proc instead? Otherwise you have to copy paste this for every tongue that has a native language, like here alone you forgot to put it on fly tongue.

Reply:
^^^^^^^^^^^^
and maybe add a linter too, SHOULD_CALL_PARENT(TRUE) will make it so children of tongues that inherit handle_speech must call parent

Reply:
> Could be a good idea to put this to tongue's handle_speech proc instead? Otherwise you have to copy paste this for every tongue that has a native language, like here alone you forgot to put it on fly tongue.

~~There's a major issue with other language. For example, aliens while speaking do hiss1-6.mp3 and with this check, they will be without hissing. And other languages have other things but just replaces of te~~

I reread you wrote, I agree, wait a bit